KomaMRIBase/src/datatypes/Sequence.jl

```diff
@@ -408,15 +408,16 @@ Returns the RF pulses and the delta frequency.
 - `Δf_rf`: (`1-row ::Matrix{Float64}`, `[Hz]`) delta frequency vector
 """
 get_rfs(seq::Sequence, t) = begin
+    ϵ = MIN_RISE_TIME
```

Comment:
Incorrect indentation
```suggestion
        ϵ = MIN_RISE_TIME
```

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/241/commits/6c2507cf6077ffacc5730b20c24184763332d45f